Libraries/PushNotificationIOS/PushNotificationIOS.js

```diff
@@ -485,6 +487,13 @@ class PushNotificationIOS {
   }
 
   /**
+   * Gets the content-available number from the `aps` object
+   */
+  getContentAvailable(): ?number {
```

Comment:
yes, it can be 1 or null/undefined so this is why it is optional. I aligned _contentAvailable member with the other private members in this class which add the optional flag only in the getter.